parambokeh/__init__.py

```diff
@@ -268,12 +268,18 @@ def action_cb(button):
 
         if hasattr(p_obj, 'callbacks'):
             p_obj.callbacks[id(self.parameterized)] = functools.partial(self._update_trait, p_name)
-        elif isinstance(w, (Button, Toggle)):
+        elif isinstance(w,  Toggle):
             if self.p.mode in ['server', 'raw']:
                 w.on_change('active', functools.partial(self.on_change, w, p_obj, p_name))
             else:
                 js_callback = self._get_customjs('active', p_name)
                 w.js_on_change('active', js_callback)
+        elif isinstance(w, Button):
+            if self.p.mode in ['server', 'raw']:
+                w.on_click(functools.partial(self.on_change, w, p_obj, p_name, None, None, None))
```

Comment:
How come you had to make that change here, does it not work without passing those ``None``s?

Reply:
Looks like those Nones might indeed be needed.

Reply:
I think they're not required for toggle because of the difference between `on_click()` vs `on_change()`. I have basically changed Button to use on_click from on_change.

Reply:
I have to admit I didn't find the actual documentation/source for on_click and on_change yet, I just guessed/saw what worked.

Reply:
Makes sense now. Thanks.